modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/ionchannelmechanismrelease/v0.1.2.json

```diff
@@ -0,0 +1,44 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/ionchannelmechanismrelease/v0.1.2/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/nexus/core/distribution/v0.1.0",
+    "{{base}}/schemas/neurosciencegraph/simulation/modelrelease/v0.1.1",
+    "{{base}}/schemas/neurosciencegraph/simulation/subcellularmodel/v0.1.1"
```

Comment:
it should be v0.1.2

Reply:
"{{base}}/schemas/neurosciencegraph/simulation/subcellularmodel/v0.1.2" shoumd be imported